web-ts/src/components/__tests__/ScrollingSearchBox.spec.tsx

```diff
@@ -55,8 +55,11 @@ describe('ScrollingSearchBox', () => {
     ).find(ScrollingSearchBox)
 
     component.find(SearchInput).prop('onFilterTextChange')('test')
-    const prop: (stickyTop: number) => void = component.find(Headroom).prop('onStickyTopChanged')
-    prop(5)
+    const prop: ((stickyTop: number) => void) | undefined = component.find(Headroom).prop('onStickyTopChanged')
+    expect(prop).toBeTruthy()
+    if (prop) {
+      prop(5)
+    }
```

Comment:
As this is only a unit test you could also tell the typescript compiler that it is defined with an exclamation mark and leave the old implementation:
```
const prop: ((stickyTop: number) => void) = component.find(Headroom).prop('onStickyTopChanged')!
prop(5)
```